### PR TITLE
Reformatting Test, Projection and TimeSeries of Transform to Width 85

### PR DESCRIPTION
**Pull request by RadicalRayan**
Guidelines followed:
-85 characters per line
-Use 4 spaces for indentation
-Dot and open parentheses stay on same line as function
-If not a preexisting line under line that we break, add an extra line after it
-Don't indent comments
-Don't break a comment if it represents output
-Don't break links
-If applicable, break right before $
-Keep math op together

Fix for issue #3478

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Projection/VectorWhiten.cs [hunk 1]
```diff
@@ -32,20 +33,29 @@ public static void Example()
...
+                column) =>
             {
-                Console.WriteLine($"{colName} column obtained post-transformation.");
+                Console.WriteLine($"{colName} column obtained " +
+                    $"post-transformation.");
+
                 foreach (var row in column)
-                    Console.WriteLine($"{string.Join(" ", row.DenseValues().Select(x => x.ToString("f3")))} ");
+                    Console.WriteLine(string.Join(" ", row.DenseValues().Select(x =>
+                        x.ToString("f3")))+" ");
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Projection/VectorWhiten.cs]: ctrl+k+d is your friend.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnn.cs [hunk 2]
```diff
@@ -102,9 +108,11 @@ public static void Example()
...
         }
 
-        private static void PrintPrediction(float value, SrCnnAnomalyDetection prediction) =>
-            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", value, prediction.Prediction[0],
-                prediction.Prediction[1], prediction.Prediction[2]);
+        private static void PrintPrediction(float value, SrCnnAnomalyDetection 
+            prediction) =>
+
+            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", value, prediction
+                .Prediction[0], prediction.Prediction[1], prediction.Prediction[2]);
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnn.cs]: should there be an empty line? pt. 1

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnnBatchPrediction.cs [hunk 3]
```diff
@@ -75,9 +82,10 @@ public static void Example()
             //5       0       0.01    0.25
         }
 
-        private static void PrintPrediction(float value, SrCnnAnomalyDetection prediction) =>
-            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", value, prediction.Prediction[0],
-                prediction.Prediction[1], prediction.Prediction[2]);
+        private static void PrintPrediction(float value, SrCnnAnomalyDetection
+            prediction) =>
+            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", value, prediction
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnnBatchPrediction.cs]: should there be an empty line? pt. 2

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectIidChangePoint.cs [hunk 4]
```diff
@@ -130,9 +138,11 @@ public static void Example()
...
         }
 
-        private static void PrintPrediction(float value, ChangePointPrediction prediction) =>
-            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}\t{4:0.00}", value, prediction.Prediction[0],
-                prediction.Prediction[1], prediction.Prediction[2], prediction.Prediction[3]);
+        private static void PrintPrediction(float value, ChangePointPrediction
+            prediction) =>
+            Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}\t{4:0.00}", value,
+                prediction.Prediction[0], prediction.Prediction[1],
+                    prediction.Prediction[2], prediction.Prediction[3]);
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectIidChangePoint.cs]: second "prediction" line too far indented?

**Pull request merged**